alvr/client_core/src/connection.rs

```diff
@@ -271,6 +271,7 @@ fn connection_pipeline(capabilities: ClientCapabilities) -> ConResult {
         stream_socket.subscribe_to_stream::<Haptics>(HAPTICS, MAX_UNREAD_PACKETS);
     let statistics_sender = stream_socket.request_stream(STATISTICS);
 
+    let mut actual_throughput_inseconds: f32 = 30_000_000.0;
```

Comment:
If I'm not mistaken this is bits per second per second, so rename to `actual_throughput_bps2`